.github/actions/session_nonzero/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Azure Nonzero session execution
+description: 'Execute a script as session >= 2.'
+
+inputs:
+  exec:
+    description: Script to execute
+    required: true
+  max-attempts:
+    description: Maximum number of attempts to acquire session > 2
```

Comment:
same here (and below, we should IMHO avoid the verbiage around session > 2). It’s just the maximum number of  connection attempts.